extras/data/E142/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+E142:
+  description: Measurements with Polarised beams
+  nnpdf_id: E142
+  y_label: A1
```

Comment:
```suggestion
  y_label: g1
```

Reply:
@toonhasenack, same as above and all below.